README.md

```diff
@@ -9,6 +9,401 @@
+
+import {createStore} from 'redux';
+import rootReducer from './reducers'
+import { Provider } from 'react-redux';
+
+import {Router, Route, IndexRoute, browserHistory} from 'react-router';
+
+import App from './components/App'
+import MoviePage from './containers/MoviePage'
+import MoviesShow from './containers/MoviesShow'
+...
+
+ReactDOM.render(
+  (<Provider store={store} >
+    <Router history={browserHistory} >
+      <Route path="/" component={App} >
+        <Route path='/movies' component={MoviePage} >
+          <Route path="/movies/new" component={MoviesNew} />
+          <Route path="/movies/:id" component={MoviesShow} />
+        </Route>
+      </Route>
+    </Router>
+  </Provider>),
+document.getElementById('container'));
+```
+Note that we **must** define our `/movies/new` route first. Why? Because otherwise, the `/:id` route handler would catch it first and assing `"new"` to be the id.
+
+Let's add a link to our Movies List to add a new movie.
+
+```javascript
+// src/components/MoviesList
+import React from 'react';
+import {Link} from 'react-router';
+import MoviesListItem from './MoviesListItem';
+
+export default (props) => {
+  const movies = props.movies;
+
+  return (
+    <div>
+      <div className='col-md-4'>
+        <ul>
+          {movies.map( (movie) => <MoviesListItem movie={movie} key={movie.id}/>)}
+        </ul>
+        <Link to="/movies/new">Add a Movie</Link>
+      </div>
+    </div>
+  )
+}
+```
+
+Now, we can easily link between our new movie list and our MovieShow component!
+
+### Redirecting
+
+Finally, it would be nice if after creating the new Movie, we could "redirect" the user back to the '/movies' route. Luckily, React Router gives us a nice interface to do this using `browserHistory`.
+
+In our `MoviesNew` component, let's import `browserHistory` and use it's `push` method to change the route.
+
+```javascript
+//src/containers/MoviesNew
+import React, { Component } from 'react';
+import {connect} from 'react-redux';
+import {addMovie} from '../actions'
+import {bindActionCreators} from 'redux';
+import { browserHistory } from 'react-router';
+
+class MoviesNew extends Component {
+
+  handleSubmit(e){
+    e.preventDefault();
+    const movie = {
+      title: this.refs.movieTitle.value
+    }
+    this.props.addMovie(movie);
+    this.refs.movieTitle.value = "";
+    browserHistory.push('/movies');
+  }
+
+  render(){
+    return (
+      <form onSubmit={this.handleSubmit.bind(this)} >
+        <input type="text" ref="movieTitle" placeholder="Add a Movie" />
+      </form>
+    )
+  }
+}
+
+function mapDispatchToProps(dispatch){
+  return {
+    addMovie: bindActionCreators(addMovie, dispatch)
+  }
+}
+
+export default connect(null, mapDispatchToProps)(MoviesNew)
+```
+
+We can use `browserHistory` to update the URL in any component lifecycle method or any event handler. Now, after submitting our form, we're sent back to the index route. Awesome!
 
 ## Resources
```

Comment:
Not sure if it's included anywhere else but https://github.com/reactjs/react-router-tutorial is a great resource. (JJ i think you told me about this)